dali/ft/filecopy.cpp

```diff
@@ -2596,6 +2625,190 @@ void FileSprayer::pushParts()
     performTransfer();
 }
 
+static StringBuffer & buildAzureStorageURIBase(IStoragePlane * storagePlane, StringBuffer & uri)
+{
+    const char *accountName = storagePlane->queryStorageApiAccount();
+    uri.appendf("https://%s", accountName);
```

Comment:
I think the storage can be striped over multiple accounts.  @jakesmith can you confirm?  (Something to revisit for the striping PR).

Reply:
There is a separate jira for adding support for striped storage.  Please see the subtasks under https://track.hpccsystems.com/browse/HPCC-27456.